extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -210,7 +202,7 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		overrideEsrbRating(params);
+		params.esrb = targeting.esrbRating || params.esrb;
```

Comment:
Don't we want to take it now always from targeting/[adContext](https://github.com/Wikia/app/blob/be0b73f3696058cacfd121efd2748d2a7239ea70/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L62)?